tools/autograd/derivatives.yaml

```diff
@@ -393,9 +393,11 @@
   self: norm_backward(grad, self - other, p, result)
   other: -norm_backward(grad, self - other, p, result)
 
+# The backward formula is done in this order to improve numerical stability
+# of the higher order derivatives, see https://github.com/pytorch/pytorch/issues/43414
```

Comment:
leave a comment why -grad*result/other can't be used (because it requires result so if result is used inplace it leads to bc breakage, right?).